deploy/README.md

```diff
@@ -12,13 +12,22 @@
   [external-dns](https://github.com/kubernetes-incubator/external-dns/)
   to manage DNS records for your Ingress specifications.
 
+## Configuration
+* Update the file ingress-controller.yaml.example following your needs and rename in ingress-controller.yaml
+* Update the file skipper.yaml.example following your needs and rename in skipper.yaml
+
 ## Install
 
+    # create a specific service account for skipper and ingress
+    % kubectl apply -f skipper_serviceaccount.yaml
+    % kubectl apply -f ingress-controller.yaml
     # install skipper an ingress http router
     % kubectl create -f deploy/skipper.yaml
     # install the controller which glues together AWS and the ingress implementation
     % kubectl create -f deploy/ingress-controller.yaml
 
+For more information regarding skipper's requirements have a look here [ingress-controller](https://github.com/zalando/skipper/blob/master/docs/kubernetes/ingress-controller.md).
```

Comment:
the link should be to https://opensource.zalando.com/skipper/kubernetes/ingress-controller/